swank/match.lisp

```diff
@@ -158,17 +158,17 @@
         ((consp pattern)
          (append
           `((consp ,key))
-          (compile-select-tests (!cs-car key) (car
```

Comment:
I'm not sure about it. Seems like you shouldn't be messing with the standard readtable like that, but it doesn't hurt to get rid of those `!` characters...